tree/dataframe/inc/ROOT/RDF/GraphUtils.hxx

```diff
@@ -62,7 +62,7 @@ private:
 
    ////////////////////////////////////////////////////////////////////////////
    /// \brief Starting from any leaf (Action, Filter, Range) it draws the dot representation of the branch.
-   std::string FromGraphLeafToDot(std::shared_ptr<GraphNode> leaf);
+   std::string FromGraphLeafToDot(const GraphNode &leaf);
```

Comment:
`const` method. And also `FromGraphActionsToDot` should be const.

Reply:
thanks, done